pkg/controller/add_deployment.go

```diff
@@ -0,0 +1,10 @@
+package controller
```

Comment:
Is it a convention in the operator to name the files `add_...`? Can't they just be `deployment.go` and `jaeger.go`?

Reply:
That's boilerplate code generated from the new operator SDK